static/js/place/chart_block.tsx

```diff
@@ -195,6 +195,7 @@ class ChartBlock extends React.Component<ChartBlockPropType> {
     // Prepare parameters for related charts.
     let unit = this.props.data.unit;
     let scaling = this.props.data.scaling;
+    const isEarth = this.props.dcid === "Earth";
```

Comment:
we actually have a constant for Earth here: https://github.com/datacommonsorg/website/blob/5e84f21c7378880b02d3a1e1e75b869ca263957f/static/js/shared/constants.ts#L35 

Instead of using the string "Earth", it will be better to use the dcid field of this constant. This applies to all other instances in this file where "Earth" is used.

Reply:
Done